plugins/module_utils/mso.py

```diff
@@ -168,6 +168,20 @@ def mso_contractref_spec():
     )
 
 
+def mso_static_path_spec():
+    return dict(
+        type=dict(type="str", choices=["port", "vpc", "dpc"]),
+        pod=dict(type="str"),
+        leaf=dict(type="str"),
+        fex=dict(type="str"),
+        path=dict(type="str"),
+        vlan=dict(type="int"),
+        primary_micro_segment_vlan=dict(type="int"),
+        deployment_immediacy=dict(type="str", choices=["immediate", "lazy"]),
+        mode=dict(type="str", choices=["native", "regular", "untagged"]),
+    )
+
+
```

Comment:
there is def mso_site_anp_epg_bulk_staticport_spec(): in mso.py with same attributes, if so can we rename it in a way so both modules can use it?

Reply:
Have changed it to use this argument spec. 

Another question that now comes to mind, is do you think we should we add a deprecation message and redirect to normal module in the bulk module now this functionality is exposed there? @lhercot, @shrsr, @sajagana, @samiib  what are your thoughts on this?

Reply:
If you have merged all the bulk functionality into the normal module and don't have plans to maintain the bulk module, then a deprecation message sounds like it could be a good idea.

Reply:
i second that

Reply:
Yeah we can put deprecation but be clear in the message that they need to use the replace attribute for the same behavior.